test/cli/simple-package/pass.package-tree.exp

```diff
@@ -0,0 +1,130 @@
+class <emptyTree><<C <root>>> < (::<todo sym>)
```

Comment:
What is this file for? `*.exp` files are only tested on testdata tests.

Reply:
Whoops. I `cp -r`'d this test from a testdata test and left all the files it came with. I'll delete this file.